styles.css

```diff
@@ -375,17 +373,58 @@ body {
 
 h1, h2, h3, h4, h5, h6 {
   font-family: "Fjalla One", sans-serif;
+  font-weight: normal;
+}
+
+h1 {
+  font-size: 5rem;
+  text-transform: uppercase;
+  line-height: 130%;
+  letter-spacing: -0.01em;
+}
+
+h2 {
+  font-size: 3rem;
+  margin-bottom: 1rem;
+}
+
+h3 {
+  font-size: 4rem;
+}
+
+h4 {
+  font-size: 3.5rem;
+}
+
+h5 {
+  font-size: 3rem;
+}
+
+h6 {
+  font-size: 2.5rem;
+}
+
+@media screen and (min-width: 700px) {
+  h1 {
+    font-size: 8rem;
+  }
+
+  h2 {
+    font-size: 5rem;
+  }
 }
 
 p {
   font-weight: 300;
+  letter-spacing: -0.06em;
+  font-size: 1.8rem;
+  line-height: 150%;
 }
 
 a,
 a:link,
 a:visited {
   color: #000;
-  text-decoration: none;
 }
 
 a:hover,
```

Comment:
We now have a link hover state that's the same as the default, aka an underline. Let's bring back the yellow background color on rollover to stay in line with our other sites and keep a distinct look when hovered.